### PR TITLE
add docs for sigstore/cosign pr3052

### PR DESCRIPTION
**Pull request by dmitris**
#### Summary
add documentation for the new command-line options to `cosign sign` related to the custom CA of the TSA server or to the mTLS connection to it.  Pull Request: https://github.com/sigstore/cosign/pull/3052, Issue: https://github.com/sigstore/cosign/issues/3006.

#### Release Note
- None

#### Documentation
- documentation new command-line parameters for `cosign sign` related to mTLS connection to the TSA Server / https://github.com/sigstore/cosign/pull/3052

### CODE REVIEW

File: content/en/cosign/timestamps.md [hunk 1]
```diff
@@ -49,6 +49,27 @@ To verify, retrieve the TSA's certificate chain, which must contain the root CA
 cosign verify --timestamp-certificate-chain ts_chain.pem <artifact>
 ```
 
+### mTLS connection to the TSA server
+`cosign sign` accept several additional optional parameters to pass the CA certificate of
```
Comment (ltagliaferri) [inline content/en/cosign/timestamps.md:53]: Add full return after heading, subject / verb agreement 

```suggestion

`cosign sign` accepts several additional optional parameters to pass the CA certificate of
```

File: content/en/cosign/timestamps.md [hunk 2]
```diff
@@ -49,6 +49,27 @@ To verify, retrieve the TSA's certificate chain, which must contain the root CA
...
+the TSA server in case it uses a custom CA or to establish mutual TLS connection to the TSA server:
```
Comment (ltagliaferri) [inline content/en/cosign/timestamps.md:54]:
```suggestion
the TSA server in cases where it uses a custom CA, or to establish a mutual TLS connection to the TSA server:
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
We'll also wait to merge based on the cosign pr

File: content/en/cosign/timestamps.md (hunk 2 above)
Comment (dmitris) [reply content/en/cosign/timestamps.md]: thanks, changed in d6dbe6d

File: content/en/cosign/timestamps.md (hunk 1 above)
Comment (dmitris) [reply content/en/cosign/timestamps.md]: done in d6dbe6d

**Review by ltagliaferri — APPROVED** [review]
Still waiting for the PR to merge but want to approve this

**Pull request merged**